internal/controller/admission/falconadmission_controller_test.go

```diff
@@ -168,6 +226,78 @@ var _ = Describe("FalconAdmission controller", func() {
 				}
 				return nil
 			}, time.Minute, time.Second).Should(Succeed())
+
+			By("Deleting FalconAdmission")
+			Eventually(func() error {
+				return k8sClient.Delete(ctx, falconAdmission)
+			})
+			Expect(err).To(Not(HaveOccurred()))
+		})
+
+		// Testing when FalconAdmission disables the admission controller
+		It("should successfully reconcile a custom resource for FalconAdmission", func() {
```

Comment:
```suggestion
		It("should successfully reconcile FalconAdmission custom resource - with admission control disabled", func() {
```
minor suggestion: Instead of adding a comment above the test description, we should make the test description, which prints to console, more descriptive so we can distinguish the 2 different tests when running them.